### PR TITLE
Liuliu/add mobile tests for profile and org pages

### PR DESCRIPTION
**Pull request by liuliu-dev**

### CODE REVIEW

File: cypress/integration/utils/devices.ts [hunk 1]
```diff
@@ -80,6 +107,18 @@ function getAppLayoutTests(tests: Tests, skipNavbar = false, loggedIn = false) {
...
 }
 
+function getAppLayoutTestsMobile(
+  tests: Tests,
+  skipNavbar = false,
+  loggedIn = false,
+) {
+  if (skipNavbar) return [...tests];
+  if (loggedIn) {
+    return [...testMobileNavbar, ...tests];
```
Comment (tbantle22) [inline cypress/integration/utils/devices.ts:117]: I think eventually we'll need different mobile navbar tests for `loggedIn` (i.e. logged in mobile navbar would have signout button and logged in wouldn't) so these would be different

File: cypress/integration/utils/devices.ts [hunk 2]
```diff
@@ -80,6 +107,18 @@ function getAppLayoutTests(tests: Tests, skipNavbar = false, loggedIn = false) {
   return [...testSignedOutNavbar, ...tests];
 }
 
+function getAppLayoutTestsMobile(
+  tests: Tests,
+  skipNavbar = false,
+  loggedIn = false,
+) {
+  if (skipNavbar) return [...tests];
```
Comment (tbantle22) [inline cypress/integration/utils/devices.ts:115]: Don't need to destructure `tests` here
```suggestion
  if (skipNavbar) return tests;
```

File: cypress/integration/utils/devices.ts [hunk 3]
```diff
@@ -72,6 +81,24 @@ export const desktopDevicesForAppLayout = (
...
+) => {
+  const desktopT = getAppLayoutTests(desktopTests, skipNavbar, loggedIn);
+  const iPadT = getAppLayoutTestsMobile(mobileTests, skipNavbar, loggedIn);
+  const iPhoneT = getAppLayoutTestsMobile(mobileTests, skipNavbar, loggedIn);
+  return [
+    macbook15(pageName, desktopT, loggedIn),
+    macbook11(pageName, desktopT, loggedIn),
+    iPad2(pageName, iPadT, loggedIn),
+    iPhoneX(pageName, iPhoneT, loggedIn),
+  ];
```
Comment (tbantle22) [inline cypress/integration/utils/devices.ts:99]: Can simplify this a bit using the functions you already have:
```suggestion
  return [
    ...desktopDevicesForAppLayout(desktopTests, skipNavbar, loggedIn)
    ...mobileDevicesForAppLayout(mobileTests, skipNavbar, loggedIn)
  ];
```

File: cypress/integration/productionTests/publicPaths/render/organizations/billing.spec.ts [hunk 4]
```diff
@@ -117,9 +117,53 @@ describe(`${pageName} renders expected components on different devices`, () => {
     ),
   ];
 
-  const tests = [...testProfileCard, ...testTabContainer];
+  const testProfileCardMobile = [
```
Comment (tbantle22) [inline cypress/integration/productionTests/publicPaths/render/organizations/billing.spec.ts:120]: For all of these organization page mobile tests, you should factor out the common tests so you don't have a bunch of repeated code. So would look something like:
```
const commonTests = [tests]
const desktopTests = [...commonTests, testsUniqueToDesktop]
const mobileTests = [...commonTests, testsUniqueToMobile]
```
Makes it easier to spot what the differences should be

**Review by tbantle22 — COMMENTED** [review]
Tests look great! Just had some comments about reusing existing code

File: cypress/integration/utils/devices.ts (hunk 1 above)
Comment (liuliu-dev) [reply cypress/integration/utils/devices.ts:117]: sounds good! so we leave it as it is for now?

Comment (tbantle22) [reply cypress/integration/utils/devices.ts:117]: Yep! Fine for now

File: cypress/integration/productionTests/privatePaths/render/profile/bounties.spec.ts [hunk 5]
```diff
@@ -30,7 +30,32 @@ describe(`${pageName} renders expected components on different devices`, () => {
       beVisible,
     ),
   ];
+
+  // skip mobile test, wait for sign out button
```
Comment (tbantle22) [inline cypress/integration/productionTests/privatePaths/render/profile/bounties.spec.ts:34]: For comments like these, if you add "TODO" to the beginning it makes it easier to search for commented out code/tests that need to be fixed

**Pull request merged**